### ISSUE TITLE
0.90.0 issue with hb_cache_host & hb_cache_path with universal creative

### ISSUE THREAD

**Issue by Austinb** [prebid/prebid-server#1130]
There is an issue with how the responses are returned from PBS regarding the values set for `hb_cache_host` and `hb_cache_path` and using the universal creative (UC.js).  In the PBS side the leading slash from `hb_cache_path` (https://github.com/prebid/prebid-server/blob/master/prebid_cache_client/client.go#L77) is now removed since 0.90.0 which causes issues on the UC.js (delivery) side of things.  Using the PBS settings of 
```yaml
external_cache:
  host: myhost.com
  path: /c/v1/cache
```

Before 0.90.0 the cache values in the `bid[i].ext.prebid` response from PBS would be something like 
```json
{
  "targeting": {
    "hb_bidder": "appnexus",
    "hb_bidder_appnexus": "appnexus",
    "hb_cache_host": "myhost.com",
    "hb_cache_host_appnex": "myhost.com",
    "hb_cache_id": "xxxxx",
    "hb_cache_id_appnexus": "xxxxxx",
    "hb_cache_path": "/c/v1/cache",
    "hb_cache_path_appnex": "/c/v1/cache",
    "hb_env": "mobile-app",
    "hb_env_appnexus": "mobile-app",
    "hb_pb": "0.00",
    "hb_pb_appnexus": "0.00",
    "hb_size": "320x100",
    "hb_size_appnexus": "320x100"
  }
}
```

Now the response comes back as
```json
{
  "targeting": {
    "hb_bidder": "appnexus",
    "hb_bidder_appnexus": "appnexus",
    "hb_cache_host": "myhost.com",
    "hb_cache_host_appnex": "myhost.com",
    "hb_cache_id": "xxxxx",
    "hb_cache_id_appnexus": "xxxxxx",
    "hb_cache_path": "c/v1/cache",
    "hb_cache_path_appnex": "c/v1/cache",
    "hb_env": "mobile-app",
    "hb_env_appnexus": "mobile-app",
    "hb_pb": "0.00",
    "hb_pb_appnexus": "0.00",
    "hb_size": "320x100",
    "hb_size_appnexus": "320x100"
  }
}
```

The issue now is that the url built by the UC.js is incorrect (invalid).  In version 1.6.0 of the UC.js it just concats the returned values from `hb_cache_host` and `hb_cache_path` and adds `https://` to the front of the string (see https://github.com/prebid/prebid-universal-creative/blob/master/src/nativeAssetManager.js#L66).  Before the response from that function in the UC.js would be something like `https://myhost.com/c/v1/cache` now the response is `https://myhost.comc/v1/cache` which basically breaks prebid everywhere using that PBS since the line item can not access the cached bid response so the ad ends up returning blank.  

Current solution is to add a `/` to the end of the `external_cache.host` which makes a PBS response look like 
```json
{
  "targeting": {
    "hb_bidder": "appnexus",
    "hb_bidder_appnexus": "appnexus",
    "hb_cache_host": "myhost.com/",
    "hb_cache_host_appnex": "myhost.com/",
    "hb_cache_id": "xxxxx",
    "hb_cache_id_appnexus": "xxxxxx",
    "hb_cache_path": "c/v1/cache",
    "hb_cache_path_appnex": "c/v1/cache",
    "hb_env": "mobile-app",
    "hb_env_appnexus": "mobile-app",
    "hb_pb": "0.00",
    "hb_pb_appnexus": "0.00",
    "hb_size": "320x100",
    "hb_size_appnexus": "320x100"
  }
}
``` 
But allows the UC.js to have the correct return url for the cache lookup.  However this change also "breaks" the url for the new `bid[i].ext.prebid.cache` response.  Urls now come back as `myhost.com//c/v1/cache?uuid=xxxxx`.  Please advise on the standard way the values for `external_cache.host` and `external_cache.path` should be set on a PBS server. Personally, I would use whatever values have been defined for those two settings as is and not manipulate them in anyway when using them in a response from the PBS.

The other option is to drop the `external_cache.host` and `external_cache.path`  from your PBS settings and roll out your own version of the UC.js with the defaults for `DEFAULT_CACHE_HOST` and `DEFAULT_CACHE_PATH` set to your cache server's information.

**Comment by SyntaxNode** [comment]
We're discussing this now. I didn't realize UC.js was relying on basic concatenation. My initial preference is to restore the behavior of including a leading slash in the path.

**Comment by SyntaxNode** [comment]
The behavior change did not intent to break the contract. We weren't aware of the reliance UC.JS had on this. Please proceed with the PR to revert the change. 

Thank you very much for finding and even more-so for fixing this issue.

**Comment by Austinb** [comment]
Pull request #1131 update and appears to test fine.  Please let me know in the PR if anything needs to be updated/changed.

**Comment by SyntaxNode** [comment]
Looks good to me. Thank you. I'll seek out a second reviewer.

**Comment by Austinb** [comment]
Fixed in b623ccc
